frontend/src/pages/ImportCkan/SelectGroupsScreen.jsx

```diff
@@ -0,0 +1,589 @@
+            Authorization: `bearer ${token}`,
+          },
+        })).json();
+      }
+    } else {
+      // Append tags to the metadata type if they dont exist
+      // Create an array of existing tags
+      const typeTagList = metadataTypes[currentGroup.openDataMetadata.metadataType]
+        .tags
+        .map((tag) => tag.tagName.toLowerCase());
+
+      /* eslint-disable no-await-in-loop */
+      for (const tag of currentGroup.tags) {
+        // If the tag isn't already added on the metadata type
+        if (!typeTagList.includes(tag.display_name.toLowerCase())) {
+          await (await fetch(`/api/MetadataType/${currentGroup.openDataMetadata.metadataType}/tag`, {
+            method: 'PUT',
+            body: JSON.stringify({
+              name: tag.display_name.toLowerCase(),
+            }),
+            headers: {
+              'Content-Type': 'application/json',
+              Authorization: `bearer ${token}`,
+            },
+          })).json();
+        }
+      }
+      /* eslint-enable no-await-in-loop */
+    }
+
+    // Create metadata
+    let metadataUuid = currentGroup.openDataMetadata.metadata;
+    if (currentGroup.openDataMetadata.strategy <= 1) {
+      const createMetadataResult = await (await fetch('/api/Metadata', {
+        method: 'PUT',
+        body: JSON.stringify({
+          description: currentGroup.notes,
+          releaseState: 1, // Released (?)
+          metadataTypeUuid,
+          municipalityName,
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `bearer ${token}`,
+        },
+      })).json();
+      metadataUuid = createMetadataResult.uuid;
+    }
+    // Insert metadata
+    /* eslint-disable no-await-in-loop */
+    for (const resource of currentGroup.resources) {
+      if (resource.openDataMetadata.ignore !== true) {
+        const payload = {
+          metadataUuid,
+          url: resource.url,
+          description: resource.description === '' ? '(No description)' : 'resource.description',
+          dataFormatMimeType: 'text/plain', // TODO start and end date
+        };
+        if (resource.openDataMetadata.hasDate) {
+          payload.startDate = resource.openDataMetadata.startDate;
+          payload.endDate = resource.openDataMetadata.endDate;
+        }
+        await fetch('/api/Metadata/url', {
+          method: 'PUT',
+          body: JSON.stringify(payload),
+          headers: {
+            'Content-Type': 'application/json',
+            Authorization: `bearer ${token}`,
+          },
+        });
+      }
+    }
+    /* eslint-enable no-await-in-loop */
+    await LoadMetadataTypes();
+    nextPage();
+    setSaving(false);
+  };
+
+  const setInsertStrategy = (strategy) => {
+    const grp = currentGroup;
+    grp.openDataMetadata.strategy = strategy;
+    updateGroup(currentIndex, grp);
+  };
+
+  const handleMetadataTypeChange = (v) => {
+    // console.log(v);
```

Comment:
Is this leftover code?
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/725557/79563689-8d819a00-80ad-11ea-8f73-842ac7aa2bdd.png)